Source/MonoGame.Extended.Tests/Shapes/RectangleFTests.cs

```diff
@@ -8,7 +8,44 @@ namespace MonoGame.Extended.Tests.Shapes
     public class RectangleFTests
     {
         [Test]
-        public void RectangleF_Intersects_Test()
+        public void ConstructorsAndProperties()
+        {
+            // Create a RectangleF from ints and from floats.
+            var intRectangleF = new RectangleF(10, 20, 64, 64);
+            var floatRectangleF = new RectangleF(7.5f, 17.5f, 62.5f, 62.5f);
+
+            // Test basic constructing
+            Assert.AreEqual(new RectangleF(10, 20, 64, 64), intRectangleF);
+            Assert.AreEqual(new RectangleF(10f, 20f, 64f, 64f), intRectangleF);
+
+            Assert.AreNotEqual(new RectangleF(10f, 20f, 64f, 64f), floatRectangleF);
+            Assert.AreNotEqual(new RectangleF(7.5f, 17.5f, 62.5f, 62.5f), intRectangleF);
+
+            Assert.AreEqual(new RectangleF(7.5f, 17.5f, 62.5f, 62.5f), floatRectangleF);
+
+            // Test int + float during construction
+            Assert.AreEqual(new RectangleF(5 + 2.5f, 10 + 7.5f, 60 + 2.5f, 60 + 2.5f), floatRectangleF);
+
+            // Test float + float during construction
+            Assert.AreEqual(new RectangleF(3.75f + 3.75f, 17.5f, 62.5f, 62.5f), floatRectangleF);
+
+            // Test sides
+            Assert.AreEqual(7.5f, floatRectangleF.Left);
+            Assert.AreEqual(7.5f + 62.5f, floatRectangleF.Right);
+            Assert.AreEqual(17.5f, floatRectangleF.Top);
+            Assert.AreEqual(17.5f + 62.5f, floatRectangleF.Bottom);
+
```

Comment:
It's generally better to have a lot of small tests vs large ones that test multiple things. If you find yourself writing a comment to describe what you're testing, that's a sign you should extract that code into a new test.